basic-javascript/project-2.spec.js

```diff
@@ -1,3 +1,142 @@
 const funcs = require('./project-2');
 
-// whoops.. there is no test suite for this file. You'll simply just have to create one :/
+describe("getBiggest", () => {
+    it("should get the biggest number", () => {
+        expect(funcs.getBiggest(9, 5)).toBe(9);
+        expect(funcs.getBiggest(3, 3)).toBe(3);
+        expect(funcs.getBiggest(3, 8)).toBe(8);
```

Comment:
expect(funcs.getBiggest('9',5)).toBe('9'); It is just ineresting to test against these cases also, because javascript does type coercion.